test/wasmBenchmarker/ctests/include/simdMandelbrot.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2023-present Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SIMD_MANDELBROT_H
+#define SIMD_MANDELBROT_H
+
+#ifndef FLOAT_SIZE
+#error define FLOAT_SIZE! (float = 4 | double = 8)
+#endif
+
+#if FLOAT_SIZE == 4
+#define TYPE float
+#elif FLOAT_SIZE == 8
+#define TYPE double
+#else
+#error FLOAT_SIZE has to be 4 or 8! (float = 4 | double = 8)
+#endif
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <wasm_simd128.h>
+
+#define WIDTH 1600
+#define HIGHT 1400
+#define N 20
+#define REAL_AXIS_SHIFT -1.8 // ~ horizontal shift
+#define IMAGINARY_AXIS_SHIFT -1.0 // ~ vertical shift
+#define ZOOM 0.0015
+
+#define byte char
+
+#define getNthBit(b, n) ((b & (1 << (7 - n))) > 0)
+
+#define clearNthBit(b, n) b = b & (0xFF - (1 << (7 - n)))
+
+#define setNthBit(b, n) b = b | (1 << (7 - n))
+
+#if FLOAT_SIZE == 4
+#define const_splat(x) wasm_f32x4_const_splat(x)
+#define gt(x, y) wasm_f32x4_gt(x, y)
```

Comment:
Arguments are not needed. Simply:
`#define gt wasm_f32x4_gt`

Reply:
Okay. Thanks!